src/prefect/server/database/orm_models.py

```diff
@@ -1304,9 +1308,9 @@ class ORMFlowRunNotificationQueue:
 
 @declarative_mixin
 class ORMVariable:
-    name: str = sa.Column(sa.String, nullable=False)
-    value: str = sa.Column(sa.String, nullable=False)
-    tags: List[str] = sa.Column(JSON, server_default="[]", default=list, nullable=False)
+    name = sa.Column(sa.String, nullable=False)
```

Comment:
Can we not suggest types here anymore?

Reply:
We cannot as far as I can tell. It causes all sorts of weird problems. I removed them all in Cloud and mypy is quite upset — the SQLAlchemy types are all new in v2 though.

Reply:
ahhh gotcha